service-front/module/Application/view/email/notify.EMAIL_LPA_REGISTRATION_WITH_NO_PAYMENT3.txt

```diff
@@ -4,11 +4,11 @@ NOTIFY TEMPLATE ID: b84aa41d-c94e-4bb7-8747-28b9d6ed0d6c
 
 Online LPA ID: ((lpaId))
 
-#((donorName))'s LPA is ready to print and sign
+#((donorName))'s ((lpaType)) LPA is ready to print and sign
 
  What you need to do next
 
-* Sign in to your account to download and print the LPA and  fee remissions form ((PTN??and each person to notify letter)) here ((viewDocsUrl)) 
+* Sign in to your account to download and print the LPA and  fee remissions form ((PTN??and each person to notify letter)) here ((viewDocsUrl))
```

Comment:
what's with the "PTN??"

Reply:
This simply only shows if PTN is set to true.  Are you wondering why there's no space?  That only reason that line changed is the trailing space got removed